diff --git a/Makefile.am b/Makefile.am
index 1634c6af..2f9b7583 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -286,7 +286,7 @@ INSTALL_SYSTEMD_TARGET = install-systemd-local
 # which will be the source folder if you're not using a separate build folder

 # Install a system service script, but don't replace an existing script
-$(INSTALL_SYSTEMD_TARGET): scripts/$(SYSTEMD_SERVICE) $(INSTALL_USER_TARGET)
+$(INSTALL_SYSTEMD_TARGET): scripts/$(SYSTEMD_SERVICE)
 	install -d $(DESTDIR)$(systemdsystemunitdir)
 	[ -e $(DESTDIR)$(systemdsystemunitdir)/shairport-sync.service ] || install -m 0644 scripts/$(SYSTEMD_SERVICE) $(DESTDIR)$(systemdsystemunitdir)/shairport-sync.service
